admin/templates/Includes/LeftAndMain_Menu.ss

```diff
@@ -1,36 +1,10 @@
 <div class="cms-menu cms-panel cms-panel-layout west" id="cms-menu" data-layout-type="border">
 	<div class="cms-logo-header north">
```

Comment:
It would make overriding the top component easier, since you could then implement only that template.

However my intention is to cut down on duplication of the low level html in overriding templates, to improve maintainability.

E.g.

```
<div class="cms-logo-header north">
	<% include LeftAndMain_MenuStatus %>
	<div>My custom north component</div>
</div>
```

Versus

```
<div class="cms-logo">
    <a href="$ApplicationLink" target="_blank" title="$ApplicationName (Version - $CMSVersion)">
		$ApplicationName <% if $CMSVersion %><abbr class="version">$CMSVersion</abbr><% end_if %>
    </a>
    <span><% if $SiteConfig %>$SiteConfig.Title<% else %>$ApplicationName<% end_if %></span>
</div>
<div class="cms-login-status">
	<a href="$LogoutURL" class="logout-link font-icon-logout" title="<%t LeftAndMain_Menu_ss.LOGOUT 'Log out' %>"></a>
	<% with $CurrentMember %>
		<span>
			<%t LeftAndMain_Menu_ss.Hello 'Hi' %>
			<a href="{$AbsoluteBaseURL}admin/myprofile" class="profile-link">
				<% if $FirstName && $Surname %>$FirstName $Surname<% else_if $FirstName %>$FirstName<% else %>$Email<% end_if %>
			</a>
		</span>
	<% end_with %>
</div>
<div>My custom north component</div>
```